lib/phoenix/live_dashboard/pages/ecto_stats_page.ex

```diff
@@ -70,27 +79,37 @@ defmodule Phoenix.LiveDashboard.EctoStatsPage do
       Enum.all?(repos, fn repo -> repo not in capabilities.processes end) ->
         :skip
 
-      extra_available_for_any?(repos) ->
+      extra_available_for_any?(Node.self(), repos) ->
         {:ok, @page_title}
 
       true ->
         {:disabled, @page_title, @disabled_link}
     end
   end
 
-  defp extra_available_for_any?(repos) do
-    Enum.any?(repos, &extra_available?/1)
+  defp extra_available_for_any?(node, repos) do
+    Enum.any?(repos, fn repo -> extra_available?(node, repo) end)
   end
 
-  defp extra_available?(repo) when is_atom(repo) do
-    extra = info_module_for(repo)
-    extra && Code.ensure_loaded?(extra)
+  defp extra_available?(node, repo) when is_atom(repo) do
+    extra = info_module_for(node, repo)
+    extra && extra_loaded?(node, extra)
   end
 
-  defp extra_available?(_repo_pid), do: false
+  defp extra_available?(_node, _repo_pid), do: false
+
+  defp extra_loaded?(node, extra) do
+    case :rpc.call(node, Code, :ensure_loaded?, [extra]) do
+      true ->
+        true
+
+      _ ->
+        false
+    end
```

Comment:
```suggestion
    :rpc.call(node, Code, :ensure_loaded?, [extra]) == true
```

Reply:
Unfortunately this means the remote node needs to have EctoPSQLExtras running in it...  what if we change EctoPSQLExtras to allow `{node, atom}` as the repo and then it automatically does a rpc call? :D If you send a PR there, please cc me. :)

Reply:
Just to update: I'm working on this PR :D